app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::CommentsController < Api::ApplicationController
+
+  def create
+    if debate
```

Comment:
Did you test it ? debate variable does not exist at all. please add a comments_controller_spec